tests/full-disk-encryption.spec.ts

```diff
@@ -27,8 +28,8 @@ test.describe('The main page', () => {
         await storagePage.validateEncryptionIsUsed();
         await storagePage.back();
 
-        await expect(page.getByText("SUSE ALP Dolomite")).toBeVisible({ timeout: 2 * minute });
-        await page.getByRole('link', { name: 'Users' }).click();
+        await expect(page.getByText(process.env.PRODUCTNAME)).toBeVisible({ timeout: 2 * minute });
```

Comment:
please, do the assertion in the Page, the test layer should not contains this long-reading lines, ie:
`expect_product_name_visible()`

Reply:
Not needed, removed.